packages/hash/frontend/src/graphql/queries/knowledge/entity.queries.ts

```diff
@@ -75,6 +75,33 @@ export const getAllLatestEntitiesQuery = gql`
   ${subgraphFieldsFragment}
 `;
 
+export const aggregateEntitiesQuery = gql`
```

Comment:
I opted to not remove getAllLatestEntities in this PR. I thought it would make sense to do it partially, and wait until we've fully implemented this query before migrating frontend code that doesn't use the hook)

Reply:
This is now redundant, I removed it at another point